src/components/picker/picker.tsx

```diff
@@ -195,28 +196,30 @@ export class Picker {
             props.maxItems = 1;
         }
 
-        return [
-            <limel-chip-set
-                type="input"
-                label={this.label}
-                leadingIcon={this.leadingIcon}
-                value={this.chips}
-                disabled={this.disabled}
-                readonly={this.readonly}
-                required={this.required}
-                searchLabel={this.searchLabel}
-                onInput={this.handleTextInput}
-                onKeyDown={this.handleInputKeyDown}
-                onChange={this.handleChange}
-                onInteract={this.handleInteract}
-                onStartEdit={this.handleInputFieldFocus}
-                onStopEdit={this.handleStopEditAndBlur}
-                emptyInputOnBlur={false}
-                clearAllButton={this.multiple}
-                {...props}
-            />,
-            this.renderDropdown(),
-        ];
+        return (
+            <Host>
```

Comment:
Is this commit still necessary? Things seem to be working fine without it as well… But I don't know if this makes things better than before or what :)

Reply:
You're right! Commit now longer needed. I prefer this syntax but since it's not needed anymore I will just remove it.

Less is more :) Care to approve again? 😄